exercicios/para-casa/dataset_alunas.py

```diff
@@ -59,4 +59,129 @@
         "Presença": [True, True, True, False, True],
         "Participação": 7.5
     }
-}
\ No newline at end of file
+}
+
+
+
+print ("-------------------------hello world!!---------------------------------")
+print ("-------------------Esse é o sistema reprograma-------------------------")
+print ("-----------------------seja bem vinde ;) ------------------------------")
+print ("----------Para ter acesso a lista de alunas (digite 1)-----------------")
+print ("----------------------Para inserir aluna nova (digite 2)--------------")
+print ("-------------------Para consultar presença (digite 3)-----------------")
+print ("----------------Para consultar notas  (digite 4)------------------------")
+print ("----------------Para calcular média (digite 5)------------------------")
+print ("-----------------Para sair do sistema (digite 6)----------------------")
+
+ 
+
+ 
+
+     
+    
+def inclusao_novas_alunas ():
+    while True:
+     nome = str(input("Insira o nome: "))
+     sobrenome = input("Insira o seu sobrenome: ")
+     turma = input(" Insira a sua turma: ")
+   
+     notas = []
+     for i in range (3):
+       nota = int(input (f"Insira as suas {i+1} notas: "))
```

Comment:
A nota de particição necessita ser float, dado que a nota é um valor decimal.

**Sugestão:**
```suggestion
       nota = float(input(f"Insira as suas {i+1} notas: "))
```